pyigm/guis/igmguesses.py

```diff
@@ -1003,6 +1015,11 @@ def on_key(self,event):
         if event.key == 'P':  # Toggle colorful mode
             self.flag_colorful = not self.flag_colorful
 
+        # print component info
+        if event.key == '@':  #  for develop; this will be deleted in future.
```

Comment:
This is only intended to be use for developers, because `blending_info` is taking too long to compute for sightlines with more than, say, 10 components which I find annoying. Still, for the patient this provides useful information so I want to keep it for now. My goal is to find out a better way to obtain this information that hopefully increases speed.

My thought are toward adding an extra attribute to components, which stores the relation of it w/r to all the rest in the sense of overlapping in wobs space. But may need to work on linetools Issue #260 before proceeding further.